python_scripts/parameter_tuning_parallel_plot.py

```diff
@@ -0,0 +1,134 @@
+# %% [markdown]
+# # Hyperparameter tuning visualization
```

Comment:
```suggestion
# # Interactive exploration of hyper-parameter search results
```

Reply:
I am just worried about changing the title here but not inside the video, though I like your suggestion the best.  Do you think it's ok to have this small difference?

Reply:
I think it's fine if the in-video title is not exactly the same.